Source/WebCore/Sources.txt

```diff
@@ -3144,6 +3144,10 @@ xml/parser/XMLDocumentParserLibxml2.cpp
 xml/parser/XMLDocumentParserScope.cpp
 
 // Derived Sources
+// FIXME: These files are individually added to WebCore_SOURCES (as they must be in order to ensure
+// they are generated before WebCore is compiled). They should not be listed here as this means they
+// are compiled twice, violating the one-definition rule. But I don't know how to remove them
+// without breaking XCode builds.
```

Comment:
This comment is wrong. The files are only being built once (unified). I'm trying to figure out why.

Reply:
tbh I'm going to give up now. I don't understand.

I'll just remove the FIXME because the right thing is happening. Everything's fine.

Reply:
OK, I found my mistake. The source files added to WebCore_SOURCES do not get compiled because `WEBKIT_COMPUTE_SOURCES()` is smart and marks them all as header files if they're included in the unified build, so their presence in WebCore_SOURCES only creates a dependency to guarantee they're generated in time and does not cause them to be compiled separately.

Sorry for spreading so much confusion.